client/post-editor/editor-media-advanced/index.jsx

```diff
@@ -0,0 +1,40 @@
+/**
+ * External depencencies
+ */
+import React, { PropTypes } from 'react';
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import { toggleEditorMediaAdvanced } from 'state/ui/editor/media/actions';
+import Dialog from 'components/dialog';
+
+function EditorMediaAdvanced( { visible, toggleVisible } ) {
+	return (
+		<Dialog isVisible={ visible } onClose={ toggleVisible }>
+			WIP
+		</Dialog>
+	);
+}
+
+EditorMediaAdvanced.propTypes = {
+	visible: PropTypes.bool,
+	toggleVisible: PropTypes.func
+};
+
+EditorMediaAdvanced.defaultProps = {
+	visible: false,
+	toggleVisible: () => {}
+};
+
+export default connect( ( state ) => {
+	return {
+		visible: state.ui.editor.media.advanced
+	};
+}, ( dispatch ) => {
+	return bindActionCreators( {
+		toggleVisible: toggleEditorMediaAdvanced
+	}, dispatch );
+} )( EditorMediaAdvanced );
```

Comment:
I agree that it would be noisy with more props being returned, but I find this _actually_ confusing now the way it is. Is it really _inconsistent_ to have these differences?

That being said, I guess the way you wrote the functions in the previous response looks better to me than the way in this PR, mainly because of the weird alignment in the PR code of the input parameters across the functions.

These style things always surprise/interest me because usually someone else feels the opposite way from how I feel :smile: To me, the consistency here is simply calling `connect()` at the end of the module and it gets two functions as per the Redux spec. If we were doing something funky like calling slightly-different connect functions then I would consider that a bigger violation of consistency.